### PR TITLE
Introduction to unity engine

### LINKED ISSUE #5767: Introduction to unity engine
### Writing sample(s): 
https://www.section.io/engineering-education/getting-started-with-game-development-windows-forms/
https://www.section.io/engineering-education/getting-started-with-inheritance-using-c/
https://www.section.io/engineering-education/understanding-polymorphism-using-abstract-classes-in-c/

#### Proposed title of article
Introduction to unity engine using c#

#### Proposed article introduction
Unity engine is one of the strongest engines in game development and it's one of the easiest engines that would help a programmer begin his journey in the gaming development industry. In this tutorial, we will explain the interface of the engine and understand how to create different objects, manipulate them and compose them together, we will also use prefabs to enhance the ability to reuse and edit different objects.

#### Key takeaways 
- engine interface
- create, edit, compose objects
- prefabs

#### Article quality
This tutorial will go in depth on how to understand the interface of the engine and will explain in detail how to work with unity objects. The reader will be able to work step by step while reading this tutorial.

#### References
https://unity.com

### PR DESCRIPTION
**Pull request by mohamedgh16**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

This fixes #5767

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/articles/introduction-to-unity-engine/index.md [hunk 1]
```diff
@@ -1,88 +1,97 @@
...
-Unity engine is one of the strongest engines in game development and it's one of the easiest engines that would help a programmer begin his journey in the gaming development industry. In this tutorial, we will explain the interface of the engine and understand how to create different objects, manipulate them and compose them together, we will also use prefabs to enhance the ability to reuse and edit different objects.
+The Unity engine is one of the strongest engines in game development, and also one of the easiest engines to learn as a beginner in game development. 
+
+In this tutorial, we will explain the interface of the engine and understand how to create different objects, manipulate them and compose them together, we will also use prefabs to enhance the ability to reuse and edit different objects.
 
 ### Table of contents
-- Understand the engine interface
-- Create edit and compose objects
-- Understand the reusability of prefabs
+- [Prerequisities](#prerequisities)
```
Comment (Jethro-magaji) [inline content/articles/introduction-to-unity-engine/index.md:9]: Each list should be named according to the corresponding H3 tag

**Comment by mohamedgh16** [comment]
@Jethro-magaji Thank you, I have done the changes👋.

**Review by Jethro-magaji — APPROVED** [review]
Hi! @mohamedgh16 
Well done for coming up with such an amazing article. Please take note of these corrections to minimize revision next time.
Thanks

**Comment by hectorkambow** [comment]
jumping on this one

**Comment by hectorkambow** [comment]
@mohamedgh16 can you please make sure there are no underscore (or capital letters) for the in article images 👍  

(best practices) 

Lets also be sure to fix the image paths afterward. 

cc @Jethro-magaji

**Comment by mohamedgh16** [comment]
@louisefindlay23 This Pr is at the end of the queue and ready for publication, can we proceed and publish it?
[This is the link of the duplicate](https://github.com/section-engineering-education/engineering-education/pull/7346).

**Pull request closed without merging**